ci/cloudbuild/fabric-peer.yaml

```diff
@@ -0,0 +1,18 @@
+steps:
```

Comment:
Correct me if I'm mistaken but it looks like we have similarities between the 3 build files, maybe we should have one config and add a ${DOCKERFILE_IMAGE} variable?

Reply:
Done